src/microbe_stage/Microbe.cs

```diff
@@ -202,22 +205,49 @@ public MicrobeSignalCommand SignalCommand
 
     /// <summary>
     ///   If true this shifts the purpose of this cell for visualizations-only
-    ///   (stops the normal functioning of the cell).
+    ///   (Completely stops the normal functioning of the cell).
     /// </summary>
     [JsonIgnore]
     public bool IsForPreviewOnly { get; set; }
 
     [JsonIgnore]
     public Spatial EntityNode => this;
 
+    [JsonIgnore]
+    public GeometryInstance EntityGraphics => Membrane;
+
+    [JsonIgnore]
+    public int RenderPriority
+    {
+        get => renderPriority;
+        set
+        {
+            renderPriority = value;
+
+            if (onReadyCalled)
+                ApplyRenderPriority();
+        }
+    }
+
     [JsonIgnore]
     public List<TweakedProcess> ActiveProcesses
     {
         get
         {
             if (processesDirty)
                 RefreshProcesses();
-            return processes!;
+            return processes;
+        }
+    }
+
+    [JsonIgnore]
+    public Dictionary<Enzyme, int> Enzymes
+    {
+        get
+        {
+            if (enzymesDirty)
+                RefreshEnzymes();
```

Comment:
Isn't there potential bug here as the organelle components only update the enzymes when they process, but this won't know when that has happened so this can read the empty lists of enzymes  from the orgnaelles?

Reply:
Or am I misremembering things? If the enzyme setup is when the organelles is attached callback, then this is actually not a problem (similarly to how the processes should work correctly even when queried too early).

Reply:
Enzyme setup is in `FinishAttachToMicrobe` specifically before the organelle components setup. The rest is exactly how `ActiveProcesses` is handled.

Reply:
So the data *is* setup basically immediately, which means that the bug I fear, should not be able to happen.